src/error.rs

```diff
@@ -85,6 +88,8 @@ impl<'tcx> fmt::Display for EvalError<'tcx> {
                 write!(f, "array index {} out of bounds {} at {:?}", index, len, span),
             EvalError::Math(span, ref err) =>
                 write!(f, "mathematical operation at {:?} failed with {:?}", span, err),
```

Comment:
Nit: I think I would use `{err} at {span}` instead of even mentioning "mathematical operation".